internal/broker/export_test.go

```diff
@@ -105,5 +108,26 @@ func (b *Broker) FetchUserInfo(sessionID string, cachedInfo *authCachedInfo) (st
 		return "", err
 	}
 
-	return b.userInfoFromClaims(uInfo, groups)
+	userInfo := b.userInfoFromClaims(uInfo, groups)
+	data, err := json.Marshal(userInfo)
+	if err != nil {
+		return "", err
+	}
+	var indented bytes.Buffer
+	err = json.Indent(&indented, data, "", "\t\t")
+	return indented.String(), err
+}
+
+var ErrTestChallengeDecodeError error
```

Comment:
I don’t think the approach is correct that way, Only integration tests should change the default, not for default tests where it should be some tests parameters.